src/Microsoft.DotNet.Interactive/PackageRestoreContext.cs

```diff
@@ -112,7 +120,7 @@ private IEnumerable<string> NativeProbingRoots()
             {
                 yield return Tuple.Create("i", rs);
             }
-            foreach (var pr in RequestedPackageReferences)
+            foreach (var pr in RequestedPackageReferences.Concat(ResolvedPackageReferences.Where(p => p.Requested).Select(p => new PackageReference(p.PackageName, p.PackageVersion))).OrderBy(r => r.PackageName))
```

Comment:
I don't understand what the `Requested` property means. If it's in this collection, wasn't it by definition "requested"?

Reply:
This is getpackagemanager lines, previously resolved nuget packages contribute to future resolutions also.

Reply:
But what does this property signify? I'm still not understanding it.

Reply:
REquested Package references are the collected set for this submission, resolve are the ones that were accumualated from successfull earlier submissions

Reply:
I'm probably still missing the point but they're in separate collections that signify this exact distinction, so why the need for a property?

Reply:
I got rid of the property, it wasn't necessary to distinguish between them after all.